uaclient/api/u/security/package_manifest/v1.py

```diff
@@ -5,7 +5,7 @@
 from uaclient.data_types import DataObject, Field, StringDataValue
 
 
-class PackageManifestResults(DataObject, AdditionalInfo):
```

Comment:
Technically this is a breaking change, but I think we're confident no one is relying on importing this class so I think we can make it.

Reply:
I kept the backwards compatibility using Alberto's suggestion